File: CSharp/modul6_kel46.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace BAB7_KEL46{
    class Program{
        class Mobil {
            private string Merk;
            private string Tipe;
            private string Warna;
            private string jenisBBM;

            public void setMerk(string val){this.Merk = val;}
            public string getMerk() {return Merk;}
            public void setTipe(string val){this.Tipe = val;}
            public string getTipe(){ return Tipe;}
            public void setWarna(string val){this.Warna = val;}
            public string getWarna(){return Warna;}
            public void setJenisBBM(string val){this.jenisBBM = val;}
            public string getJenisBBM(){return jenisBBM;}
        }
        static void Main(string[] args)
        {
            Mobil mobilSaya = new Mobil();
            mobilSaya.setMerk("Suzuki");
            mobilSaya.setTipe("Ertiga");
            mobilSaya.setWarna("Hitam");
            mobilSaya.setJenisBBM("Pertamax");

            Console.WriteLine( "Saya punya mobil bermerk " + mobilSaya.getMerk() +
                " dengan tipe " + mobilSaya.getTipe() +
                ".\nMobil saya berwarna " + mobilSaya.getWarna() +
                ".\nMobil saya menggunakan bensin berjenis " + mobilSaya.getJenisBBM()
                );

            Console.ReadLine();
        }
    }
}

File: Java/modul6_kel46.java
public class modul6_kel46 {

    public static void main(String[] args) {
        
        //membuat objek mobil
        modul6_kel46_settergetter mobil_baru = new modul6_kel46_settergetter();
        
        //mengatur nilai atribut
        mobil_baru.setMerk("Mitsubishi");
        mobil_baru.setTipe("Mirage");
        mobil_baru.setWarna("merah");
        mobil_baru.setJenisBBM("Pertamax");
        
        //mencetak nilai
        System.out.println("Saya punya mobil dengan merk " + mobil_baru.getMerk());
        System.out.print(mobil_baru.getMerk() + " " + mobil_baru.getTipe());
        System.out.println(" berwarna " + mobil_baru.getWarna());
        System.out.println("Mobil saya menggunakan bensin berjenis " + mobil_baru.getJenisBBM() + ".");    
}
}
